webapp/shop/cred/views.py

```diff
@@ -218,6 +218,16 @@ def cred_sign_up(**_):
 
     service = build("sheets", "v4", credentials=credentials)
 
+    def extract_json_comment(obj):
+        values = []
+        for key, value in obj.items():
+            if isinstance(value, dict):
+                json_dict = json.dumps(value)
+                values.append(json_dict)
+            else:
+                values.append(value)
+        return values
+
     sheet = service.spreadsheets()
     sheet.values().append(
         spreadsheetId="1i9dT558_YYxxdPpDTG5VYewezb5gRUziMG77BtdUZGU",
```

Comment:
@usamabinnadeem-10 , for the staging sheet, the `spreadsheetId` will remain the same, but the `range` changes from `Sheet1` to `Staging` according to the sheet names in:
https://docs.google.com/spreadsheets/d/1i9dT558_YYxxdPpDTG5VYewezb5gRUziMG77BtdUZGU/edit?gid=0#gid=0